npm/builder.js

```diff
@@ -481,8 +479,8 @@ Builder.prototype.compile = function() {
 
   log.title('copy resources');
   log.debug('copy asciidoctor.css');
-  // FIXME: Find a more robust implementation to get the Asciidoctor gem install path
-  var asciidoctorGemPath = child_process.execSync("ruby -e \"print (Gem::Specification.find_by_name 'asciidoctor').full_gem_path\"");
+  // REMIND: Asciidoctor is now a Git submodule
+  var asciidoctorGemPath = 'asciidoctor';
```

Comment:
@mojavelinux Could we also remove git submodule on Asciidoctor extensions lab ? (ie. publish a version 0.1.0 on GitHub ?).
I think moving each extension into a separate repository will be too much work.